romancal/lib/engdb/tests/test_engdb_tools.py

```diff
@@ -0,0 +1,119 @@
+"""
+Test suite for engdb_tools
+
+Notes
+-----
+This file has been named specifically so it is not
+automatically found by py.test. This is because, to test,
+a connection to the internal engineering service is needed,
+which is generally not available.
+"""
+
+import logging
+import os
+
+import pytest
+from astropy.time import Time
+
+from romancal.lib.engdb import engdb_tools
+
+# Configure logging
+log = logging.getLogger(__name__)
+
+GOOD_MNEMONIC = "OPE_SCF_DIR"
+GOOD_STARTTIME = "2027-02-23T01:00:00"
+GOOD_ENDTIME = "2027-02-23T01:01:00"
+
+SHORT_STARTTIME = "2027-02-23T01:00:30"
+
+BAD_MNEMONIC = "No_Such_MNEMONIC"
+NODATA_STARTIME = "2014-01-01"
+NODATA_ENDTIME = "2014-01-02"
+
+
+def test_environmental_bad(environ_jail):
```

Comment:
```suggestion
def test_environmental_bad(environ_jail):
```
Would you update this to use monkeypatch?
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setenv
I'm not sure if it's the cause of the stpreview errors but using monkeypatch would allow removal of `environ_jail`.